app/javascript/packages/badge-tooltip/badge-tooltip-element.ts

```diff
@@ -0,0 +1,84 @@
+import { tooltip } from '@18f/identity-design-system';
+
+class BadgeTooltipElement extends HTMLElement {
+  badge: HTMLElement;
+
+  connectedCallback() {
+    this.badge = this.querySelector('.usa-tooltip')!;
+
+    this.setUpTooltip();
+    this.badge.addEventListener('mouseover', () => this.handleHover());
+    document.addEventListener('keyup', (event) => this.#handleKeyDown(event));
+  }
+
+  /**
+   * Retrieves the text to be shown in the tooltip.
+   */
+  get tooltipText(): string {
+    return this.getAttribute('tooltip-text')!;
+  }
+
+  /**
+   * Initializes the tooltip element.
+   */
+  setUpTooltip() {
+    const { tooltipBody } = tooltip.setup(this.badge);
+
+    // A default USWDS tooltip will always be visible when the badge is hovered over.
+    // To ensure the tooltip content is read when made visible,
+    // change its contents to a live region.
+    tooltipBody.setAttribute('aria-live', 'polite');
+  }
+
+  /**
+   * Handles the badge mouseover.
+   */
+  handleHover() {
+    this.showTooltip();
+  }
+
+  #handleKeyDown(event: KeyboardEvent) {
+    switch (event.key) {
+      case 'Tab':
+        if (document.activeElement === this.badge) {
+          this.showTooltip();
+        }
+        break;
+
+      default:
+    }
+  }
```

Comment:
ref https://github.com/18F/identity-idp/pull/10823#discussion_r1647950221

Reply:
Can we use [`focus`](https://developer.mozilla.org/en-US/docs/Web/API/Element/focus_event) and [`blur`](https://developer.mozilla.org/en-US/docs/Web/API/Element/blur_event) events? May be simpler and more reliable that way.